test/onnx/test_op_consistancy.py

```diff
@@ -0,0 +1,363 @@
+# Owner(s): ["module: onnx"]
+
+"""Test consistency between torch.onnx exported operators and aten operators.
+
+NOTE:
+
+When new ops are supported, please scroll down to modify the EXPECTED_SKIPS_OR_FAILS and
+ALLOWLIST_OP lists.
+
+"""
+import contextlib
+import copy
+import dataclasses
+import io
+import itertools
+import unittest
+from collections import namedtuple
+from typing import Callable, Collection, Iterable, Optional, Sequence, Tuple, Union
+
+import onnx
+
+import torch
+from torch.onnx import _constants, verification
+from torch.testing._internal import (
+    common_device_type,
+    common_methods_invocations,
+    common_utils,
+)
+from torch.testing._internal.opinfo import core as opinfo_core
+
+# The min onnx opset version to test for
+MIN_ONNX_OPSET_VERSION = 9
```

Comment:
Can the op version be shared with test_pytorch_onnx_onnxruntime.py?

Reply:
I was hesitant to import test_pytorch_onnx_onnxruntime from another test. Do you have any suggestions?

Reply:
I see. I guess each test has their own required test version making snese.